assets/js/modules/tagmanager/components/common/FormInstructions.js

```diff
@@ -115,7 +119,7 @@ export default function FormInstructions() {
 
 	return (
 		<Fragment>
-			{ hasExistingTag && (
+			{ showExistingTagMessage && hasExistingTag && (
```

Comment:
We should show the existing tag message in the settings form at the top. Please refer to this [comment](https://github.com/google/site-kit-wp/issues/4934#issuecomment-1100218663).